scripts/linters/pylint_extensions.py

```diff
@@ -2152,11 +2201,12 @@ def _check_exceptional_type_is_documented(
             line_num: int. The line number of given token.
             exceptional_type: str. The exceptional type for which this method
                 is called, Possible values can be 'Any' or 'object'.
+            node: astroid.scoped_nodes.Module. Node to access module content.
         """
         # Here, we are defining new variables so that we don't have to fetch
         # the values from 'type_status_dict' dictionary again and again.
-        outside_function_signature_block = type_status_dict[
-            'outside_function_signature_block'
+        is_function_signature_block = type_status_dict[
```

Comment:
Don't start boolean variables with verbs.

Not sure what this means, too, in the context of this function. What is it that "is a function signature block"? Do you mean in_function_signature_block, perhaps? Maybe add a comment to explain what this variable represents exactly.

Reply:
reverted to previous naming.